src/components/InventoryHostStaleness/HostStalenessCard.js

```diff
@@ -0,0 +1,188 @@
+/* eslint-disable react/no-unescaped-entities */
+import React, { useState } from 'react';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Flex,
+  Modal,
+  Popover,
+  Tab,
+  TabTitleText,
+  Tabs,
+  Title,
+} from '@patternfly/react-core';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+import TabCard from './TabCard';
+import { CONVENTIONAL_TAB_TOOLTIP, IMMUTABLE_TAB_TOOLTIP } from './constants';
+import { InventoryHostStalenessPopover } from './constants';
+// import { groupsApi } from '../../api';
+
+const HostStalenessCard = () => {
+  //multiply these values be seconds at the end before sending to the api
+  const [filter, setFilter] = useState({
+    system_staleness_delta: '2',
+    system_stale_warning_delta: '14',
+    system_culling_delta: '21',
+    edge_staleness_delta: '7',
+    edge_stale_warning_delta: '150',
+    edge_culling_delta: '180',
+  });
+
+  const [newFormValues, setNewFormValues] = useState(filter);
+  const [edit, setEdit] = useState(false);
+  const [activeTabKey, setActiveTabKey] = useState(0);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isFormValid, setIsFormValid] = useState(true);
+  // const [groupTotal, setGroupTotal] = useState(0);
+
+  const handleTabClick = (_event, tabIndex) => {
+    setActiveTabKey(tabIndex);
+  };
+  const handleModalToggle = () => {
+    setIsModalOpen(!isModalOpen);
+  };
+
+  const resetToOriginalValues = () => {
+    setNewFormValues(filter);
+    setEdit(!edit);
+  };
+
+  const updateHost = () => {};
+
+  // useEffect(() => {
+  //   groupsApi.apiGroupGetGroupList().then((res) => setGroupTotal(res.total));
+  // }, []);
+  // https://console.redhat.com/api/inventory/v1/groups get the count
+  // Need to update the edit button so that it makes the POST req,
+  return (
+    <Card id={'HostStalenessCard'}>
+      <CardHeader>
+        <Title headingLevel="h4" size="xl" id="HostTitle">
+          Organization level system staleness and culling
+        </Title>
+        <InventoryHostStalenessPopover />
+      </CardHeader>
+      <CardBody>
+        <p>
+          Keep or customize your organization's default settings using the
+          options below.
+        </p>
+        <Flex className="pf-u-mt-md">
+          <Title headingLevel="h6">System configuration </Title>
+          <a
+            onClick={() => {
+              setEdit(!edit);
+            }}
+          >
+            Edit
+          </a>
```

Comment:
```suggestion
          <a
            role="button"
            onClick={() => {
              setEdit(!edit);
            }}
          >
            Edit
          </a>
```

So that your link is treated as a button and the test passes.